sensor/kubernetes/listener/resources/dispatcher.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/stackrox/rox/sensor/common/awscredentials"
 	"github.com/stackrox/rox/sensor/common/config"
 	"github.com/stackrox/rox/sensor/common/metrics"
+	"github.com/stackrox/rox/sensor/kubernetes/complianceoperator/dispatchers"
```

Comment:
Before we merge this it should be determined if the refactor PR is going to happen or not.  If it isn't, we should put the remediation dispatcher with the rest of them.

Reply:
Now they are consistent, merged the PR to move the listeners: https://github.com/stackrox/stackrox/pull/10327